src/domain/services.ts

```diff
@@ -0,0 +1,255 @@
+* @param {APIErrorObject} error Error details
+* @param {object} span optional request span
+* @throws {FSPIOPError} Will throw an error if no endpoint to forward the services requests is
+*  found, if there are network errors or if there is a bad response
+* @returns {Promise<void>}
+*/
+export async function forwardServicesServiceTypeRequestError (
+  path: string,
+  headers: HapiUtil.Dictionary<string>,
+  serviceType: string,
+  error: APIErrorObject,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  span?: any): Promise<void> {
+  const childSpan = span?.getChild('forwardServicesServiceTypeRequestError')
+  const sourceDfspId = headers[Enum.Http.Headers.FSPIOP.SOURCE]
+  const destinationDfspId = headers[Enum.Http.Headers.FSPIOP.DESTINATION]
+  const endpointType = Enum.EndPoints.FspEndpointTypes.TP_CB_URL_SERVICES_PUT_ERROR
+
+  try {
+    const url = await Util.Endpoints.getEndpointAndRender(
+      Config.ENDPOINT_SERVICE_URL,
+      destinationDfspId,
+      endpointType,
+      path,
+      { ServiceType: serviceType }
+    )
+    Logger.info(`services::forwardServicesServiceTypeRequestError - Forwarding services error callback to endpoint: ${url}`)
+
+    await Util.Request.sendRequest(
+      url,
+      headers,
+      sourceDfspId,
+      destinationDfspId,
+      Enum.Http.RestMethods.PUT,
+      error,
+      Enum.Http.ResponseTypes.JSON,
+      childSpan
+    )
+
+    Logger.info(`services::forwardServicesServiceTypeRequest - Forwarded services error callback: ${serviceType}
+    from ${sourceDfspId} to ${destinationDfspId}`)
+    if (childSpan && !childSpan.isFinished) {
+      childSpan.finish()
+    }
+  } catch (err) {
+    Logger.error(`services::forwardServicesServiceTypeRequestError - Error forwarding services error to endpoint:
+    ${inspect(err)}`)
+    const fspiopError: FSPIOPError = ReformatFSPIOPError(err)
+    if (childSpan && !childSpan.isFinished) {
+      await finishChildSpan(fspiopError, childSpan)
+    }
+    throw fspiopError
+  }
+}
+
+/**
+* @function forwardGetServicesServiceTypeRequestToProviderService
+* @description Forwards a GET /services/{ServiceType} request to the Service Provider micro-service
+* @param {string} path Callback endpoint path
+* @param {HapiUtil.Dictionary<string>} headers Headers object of the request
+* @param {RestMethodsEnum} method The http method GET
+* @param {string} serviceType the ServiceType of the service resource
+* @param {object} span optional request span
+* @throws {FSPIOPError} Will throw an error if no endpoint to forward the services requests is
+*  found, if there are network errors or if there is a bad response
+* @returns {Promise<void>}
+*/
+export async function forwardGetServicesServiceTypeRequestToProviderService (
+  path: string,
+  headers: HapiUtil.Dictionary<string>,
+  method: RestMethodsEnum,
+  serviceType: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  span?: any): Promise<void> {
```

Comment:
using unknown in place of any allows you to drop eslint exception
```suggestion
  span?: unknown): Promise<void> {
```

Reply:
I believe the function `getChild` doesn't work on `unknown` type and thats why it's `any`. I think I need to figure out where the Span class originates from and see if they have a types library.